### PR TITLE
Scatterplot tidying

### PR DESCRIPTION
**Pull request by maxwellfundi**
These are the @bethanclarke  recommendations to improve theScatterplot

### CODE REVIEW

File: instat/dlgPlot.vb [hunk 1]
```diff
@@ -98,4 +101,21 @@ Public Class dlgPlot
...
+        Dim clsTempOp As New ROperator
+        Dim clsTempRFunc As New RFunction
+
+        If chkPoints.Checked = True Then
+            clsTempOp.SetOperation("+")
+            clsTempOp.SetParameter(True, clsRFunc:=clsRggplotFunction)
+            clsTempOp.SetParameter(False, clsRFunc:=clsRgeom_lineplotFunction)
+            clsTempRFunc.SetRCommand("geom_point")
+            ucrBase.clsRsyntax.SetOperatorParameter(True, clsOp:=clsTempOp)
+            ucrBase.clsRsyntax.SetOperatorParameter(False, clsRFunc:=clsTempRFunc)
```
Comment (dannyparsons) [inline instat/dlgPlot.vb]: Remember than an operator can have additional parameters. So you need to be make these into two operators.
You can just do `AddOperatorParameter` for the `geom_point`, without having to do anything with the `ggplot `or `geom_line`.

**Comment by dannyparsons** [comment]
We are having discussions on plots with David and Beth and there might be more changes for these plots, so I'll leave this open for now and let you know after discussions.

File: instat/dlgPlot.vb (hunk 1 above)
Comment (dannyparsons) [inline instat/dlgPlot.vb]: Do you understand my suggestion here? We can discuss if needed.

**Comment by maxwellfundi** [comment]
@dannyparsons i have doneas you said but i am unable to remove the geom_point when the check box is uchecked. could you help me with this

File: instat/dlgPlot.vb [hunk 2]
```diff
@@ -98,4 +101,14 @@ Public Class dlgPlot
     Private Sub ucrBase_ClickReset(sender As Object, e As EventArgs) Handles ucrBase.ClickReset
         SetDefaults()
     End Sub
+
+    Private Sub chkPoints_CheckedChanged(sender As Object, e As EventArgs) Handles chkPoints.CheckedChanged
+        Dim clsTempRFunc As New RFunction
+        If chkPoints.Checked = True Then
+            clsTempRFunc.SetRCommand("geom_point")
+            ucrBase.clsRsyntax.AddOperatorParameter(True, clsRFunc:=clsTempRFunc)
```
Comment (dannyparsons) [inline instat/dlgPlot.vb]: The first argument here is the name of the parameter. It should be a string, not True. Just give it a name like "geom_point".

File: instat/dlgPlot.vb [hunk 3]
```diff
@@ -98,4 +101,14 @@ Public Class dlgPlot
...
+        Else
```
Comment (dannyparsons) [inline instat/dlgPlot.vb]: In here use RemoveOperatorParameter("geom_point") <- the name you gave to this parameter.

**Comment by maxwellfundi** [comment]
@dannyparsons  this is done

**Comment by dannyparsons** [comment]
Perfect

**Pull request merged**